<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Payment</class>
 <widget class="QWidget" name="Payment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="PaymentType">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select  Payment Type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="paymentType">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>60</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Cash</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Card</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EasyPaisa</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>JazzCash</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="placeOrder">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Place Order</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="cartList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>541</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="orderTotal">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>540</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>540</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total   =</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardnum">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>170</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cvv">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>230</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="exp">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>310</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lcardnum">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>170</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Card Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="lexp">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>310</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Expiry Date(MM/YY)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lcvv">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CVV</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="laccnum">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>190</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="accnum">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>200</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Account Number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
